test/SeqCli.EndToEnd/Support/TestConfiguration.cs

```diff
@@ -30,7 +30,10 @@ public CaptiveProcess SpawnServerProcess(string storagePath)
             if (storagePath == null) throw new ArgumentNullException(nameof(storagePath));
 
             var commandWithArgs = $"run --listen=\"{ServerListenUrl}\" --storage=\"{storagePath}\"";
-            
+            if (Environment.GetEnvironmentVariable("USE_DOCKER_SEQ") == "Y")
+            {
+                return new CaptiveProcess("docker", "run --name seq -it --rm -e ACCEPT_EULA=Y -p 9989:80 datalust/seq:latest", disposeFullExePath: "docker", disposeArgs: "stop seq");
```

Comment:
Since `ServerListenUrl` is declared above with 9989 embedded in it, perhaps we could extract a `ServerListenPort` constant, and use that both here and in the definition of `ServerListenUrl`?